app/assets/src/api/constants.js

```diff
@@ -0,0 +1,4 @@
+// Names may change as schema is developed
```

Comment:
I recommend providing some more guidance on naming conventions here. For example, should we follow the convention of "noun_verb" as in SAMPLE_REPORT_VIEW? Should we elaborate on the source component names such as `SampleView.jsx` -> `SAMPLE_REPORT_VIEW` or keep consistent as in `SampleView.jsx` -> `SAMPLE_VIEW`?

Reply:
Thanks, added some guidelines. Up for discussion if desired